src/main/java/com/assignment/Assignment3.java

```diff
@@ -0,0 +1,8 @@
+package com.assignment;
+
+public class Assignment3 {
+
+    public int returnCharacterIndex(String originalString, char a) {
```

Comment:
The naming convention for a method should be :
getMethod-name -> if its returning something
setMethod-name -> if its setting something